pyproject.toml

```
@@ -41,7 +41,7 @@ Homepage = "https://github.com/lsmo-epfl/aiida-lsmo"
 [project.optional-dependencies]
 testing = [
     "pgtest~=1.3.1",
-    "aiida-testing @  git+https://github.com/ltalirz/aiida-testing@4212ba83896693ea13908eeb233ca3b35d504146", # egg=aiida-testing
+    "aiida-testing @  git+https://github.com/aiidateam/aiida-testing", # egg=aiida-testing
```

I think you currently need to use the develop branch, like
```suggestion
    "aiida-testing @  git+https://github.com/aiidateam/aiida-testing"@develop, # egg=aiida-testing
```

---

thanks @chrisjsewell